src/main/scala/nl.knaw.dans.dd.prepub/Configuration.scala

```diff
@@ -44,7 +44,7 @@ object Configuration {
     new Configuration(
       version = (home / "bin" / "version").contentAsString.stripLineEnd,
       serverPort = properties.getInt("daemon.http.port"),
-      pidGeneratorBaseUrl = properties.getString("pid-generator.base-url"),
+      pidGeneratorBaseUrl = new URI(properties.getString("pid-generator.base-url") + "/"),
```

Comment:
Add a "/" so that trailing slash becomes optional. Try it out in a worksheet:

```scala
import java.net.URI
val uri = new URI("http://test.com/blah//")
val uri2 = new URI("http://test.com/blah/")
uri resolve "create"
uri2 resolve "create"
```